Task7_1/HttpModule.Task7.SiteDownloader/Services/HttpClientFactory.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Net.Http;
+using HttpModule.Task7.SiteDownloader.Services.Abstractions;
+
+namespace HttpModule.Task7.SiteDownloader.Services
+{
+    public class HttpClientFactory : IHttpClientFactory
+    {
+        public HttpClient Create()
+        {
```

Comment:
I expect user agent header b/c with it a plenty of sites are sending you 403 in case of absence it
The value for user agent could be grabbed from from configuration file.

Reply:
Done